system/kube-monitoring/charts/ipmi-exporter/templates/deployment.yml

```diff
@@ -25,6 +25,12 @@ spec:
           value: "{{ .Values.global.ipmi_username | default .Values.ipmi_username }}"
         - name: IPMI_PASSWORD
           value: "{{ .Values.global.ipmi_password | default .Values.ipmi_password }}"
+        {{- if .Values.global.ipmi_exporter.netbox.enabled }}
+        - name: NETBOX_CP_USER
+          value: "{{ .Values.global.ipmi_username_cp | default .Values.ipmi_username_cp }}"
+        - name: NETBOX_CP_PASSWORD
+          value: "{{ .Values.global.ipmi_password_cp | default .Values.ipmi_password_cp }}"
```

Comment:
What will this default to `.Values.ipmi_password_cp`? I don't see a reference in the value files here.

Maybe it would be an idea to do:

```
{{ require "ipmi_password_cp needs to be defined in global values" .Values.global.ipmi_password_cp }}
```

Deployment will then just bomb out if it is missing instead of inserting something undefined.